pages/templates/elements.py

```diff
@@ -175,10 +193,10 @@ def select(self, item_selector):
             item_loc = (by.XPATH, f"//*[.='{item_selector}']")
             item = self.driver.find_element(*item_loc)
 
-        if item.tag_name is not 'a': 
+        if item.tag_name != 'a': 
 
             parent = item.find_element_by_xpath('..')            
-            if parent is 'a': item = parent
+            if parent == 'a': item = parent
 
```

Comment:
I did on my first commit of sign in page. This does not affect performance but my VsCode says its  an error. I should have branched this off, my bad. I don't know if you want this change implemented to be considerate to other IDEs

Reply:
It says it's an error but it still runs? What error does it give you?

Reply:
SyntaxWarning: "is not" with a literal. Did you mean "!="?
SyntaxWarning: "is" with a literal. Did you mean "=="?

I shouldn't have called it an error I should have called it a warning.

Reply:
The change is fine. keep it.